.gitignore

```diff
@@ -0,0 +1,3 @@
+The files with following extension should be ignored.
+
+*.pyc
```

Comment:
instead of this you can add `__pycache__` which will ignore all python compiled files